traffic_ops/testing/api/v5/topologies_test.go

```diff
@@ -470,42 +469,42 @@ func TestTopologies(t *testing.T) {
 					topology := testCase.RequestBody
 
 					switch method {
-					case "GET", "GET AFTER CHANGES":
-						t.Run(name, func(t *testing.T) {
-							resp, reqInf, err := testCase.ClientSession.GetTopologies(testCase.RequestOpts)
-							for _, check := range testCase.Expectations {
-								check(t, reqInf, resp.Response, resp.Alerts, err)
-							}
-						})
+					//case "GET", "GET AFTER CHANGES":
+					//	t.Run(name, func(t *testing.T) {
+					//		resp, reqInf, err := testCase.ClientSession.GetTopologies(testCase.RequestOpts)
+					//		for _, check := range testCase.Expectations {
+					//			check(t, reqInf, resp.Response, resp.Alerts, err)
+					//		}
+					//	})
 					case "POST":
 						t.Run(name, func(t *testing.T) {
 							resp, reqInf, err := testCase.ClientSession.CreateTopology(topology, testCase.RequestOpts)
 							for _, check := range testCase.Expectations {
 								check(t, reqInf, resp.Response, resp.Alerts, err)
 							}
 						})
-					case "PUT":
-						t.Run(name, func(t *testing.T) {
-							if _, ok := testCase.RequestOpts.QueryParameters["name"]; !ok {
-								t.Fatalf("Query Parameter: \"name\" is required for PUT method tests.")
-							}
-							resp, reqInf, err := testCase.ClientSession.UpdateTopology(testCase.RequestOpts.QueryParameters["name"][0], topology, testCase.RequestOpts)
-							for _, check := range testCase.Expectations {
-								check(t, reqInf, resp.Response, resp.Alerts, err)
-							}
-						})
-					case "DELETE":
-						t.Run(name, func(t *testing.T) {
-							if _, ok := testCase.RequestOpts.QueryParameters["name"]; !ok {
-								t.Fatalf("Query Parameter: \"name\" is required for DELETE method tests.")
-							}
-							alerts, reqInf, err := testCase.ClientSession.DeleteTopology(testCase.RequestOpts.QueryParameters["name"][0], testCase.RequestOpts)
-							for _, check := range testCase.Expectations {
-								check(t, reqInf, nil, alerts, err)
-							}
-						})
+					//case "PUT":
```

Comment:
This test should either not be commented out or the the commented out line should be removed.